firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -627,20 +625,18 @@ extension BrowserViewController: WKNavigationDelegate {
         }
 
         if !(url.scheme?.contains("firefox") ?? true) {
-            showSnackbar(forExternalUrl: url, tab: tab) { isOk in
-                guard isOk else { return }
-                UIApplication.shared.open(url, options: [:]) { openedURL in
-                    // Do not show error message for JS navigated links or 
-                    // redirect as it's not the result of a user action.
-                    if !openedURL, navigationAction.navigationType == .linkActivated {
-                        let alert = UIAlertController(
-                            title: .UnableToOpenURLErrorTitle,
-                            message: .UnableToOpenURLError,
-                            preferredStyle: .alert
-                        )
-                        alert.addAction(UIAlertAction(title: .OKString, style: .default, handler: nil))
-                        self.present(alert, animated: true, completion: nil)
-                    }
+            // Try to open the custom scheme URL, if it doesn't work we show an error alert
```

Comment:
We are changing our behavior here. We try to open the URL by default. If it doesn't work we show the error. So no more custom alert before we try to open.